Week 2 - Arrays/readability/readability.c

```diff
@@ -0,0 +1,56 @@
+#include <cs50.h>
+#include <ctype.h>
+#include <math.h>
+#include <stdio.h>
+
+int getLevel(string text);
+
+int main(void)
+{
+    string text = get_string("Text: ");
+
+    int level = getLevel(text);
+
+    if (level < 1)
+    {
+        printf("Before Grade 1\n");
+    }
+    else if (level >= 16)
+    {
+        printf("Grade 16+\n");
+    }
+    else
+    {
+        printf("Grade %i\n", level);
+    }
+}
+
+int getLevel(string text)
+{
+    int letters = 0;
+    int words = 0;
+    int sentences = 0;
+    int length = 0;
+
+    while (text[length] != 0)
+    {
+        if (text[length] == ' ')
```

Comment:
I would rather introduce a local variable within this loop for sake of readability
```suggestion
        char character = text[length]
        if (text[length] == ' ')
```